website/docusaurus.config.js

```diff
@@ -49,6 +49,17 @@ module.exports = {
         releaseDate: "2021-03-12",
         firstDocPath: "/getting-started",
       },
+      {
+        name: "APISIX™ Helm Charts",
+        nameInParamCase: "helm-charts",
+        description: "An Apache APISIX Helm Charts provide the installation of Apache APISIX components for kubernetes.",
+        shape: "hexagon",
+        color: "#2563EB",
+        githubRepo: "apache/apisix-helm-chart",
+        version: "0.4.0",
+        releaseDate: "2021-03-12",
+        firstDocPath: "/apisix",
+      },
```

Comment:
Hi @juzhiyuan 

Could you help me to check this, I adjust the file path and structure, but I still not found the **helm-chart** page.

Reply:
@stu01509 Hi, the docs will only appear in production env for now :)

Reply:
OK, but could you confirm the shape, color, and releaseDate? because I didn't found the release date in the repo.

Reply:
ok, this need @tokers's confirm